neqo-transport/src/connection/tests/stream.rs

```diff
@@ -111,6 +113,121 @@ fn transfer() {
     assert!(fin3);
 }
 
+#[cfg(test)]
+#[derive(PartialEq, Eq, PartialOrd, Ord)]
+struct IdEntry {
+    sendorder: StreamOrder,
+    stream_id: StreamId,
+}
+
+#[cfg(test)]
+// tests stream sendorder priorization
+fn sendorder_test(order_of_sendorder: &Vec<StreamOrder>) {
+    let mut client = default_client();
+    let mut server = default_server();
+    connect_force_idle(&mut client, &mut server);
+
+    qdebug!("---- client sends");
+    // open all streams and set the sendorders
+    let mut ordered = BinaryHeap::new();
+    let mut streams = Vec::<StreamId>::new();
+    for sendorder in order_of_sendorder {
+	streams.push(client.stream_create(StreamType::UniDi).unwrap());
+	let id = *(streams.last()).unwrap();
```

Comment:
I think that you want this instead:

```rust
let id = client.stream_create(StreamType::UniDi).unwrap();
streams.push(id);
```

Reply:
Yes.   I had the first, and then needed the id... fixed.